app/javascript/store/createStore.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from "@reduxjs/toolkit";
+import greetingReducer from "./greetingReducer";
+
+const store = configureStore({
+  reducer: {
+    greeting: greetingReducer,
+  },
+});
+
+export default store;
```

Comment:
- [ ] Nice work creating the store but also you need to connect it with your react component. you will user react-redux for that.